src/dev-app/card/card-demo.html

```diff
@@ -59,11 +59,11 @@
 
   <mat-card [appearance]="appearance">
     <mat-card-header>
-      <img mat-card-avatar>
+      <img mat-card-avatar alt="">
```

Comment:
An empty alt isn't particularly useful. It should describe what the image is about.

Reply:
Hii @crisbeto updated the alt description.
Thanks